src/main/java/seedu/foodrem/model/item/ItemExpiryDate.java

```diff
@@ -54,51 +47,22 @@ public boolean isNotSet() {
     }
 
     /**
-     * Returns true if the given datetime object is after the expiry date.
-     *
-     * @param datetime The datetime object to compare against
-     */
-    public boolean isAfterExpiryDate(LocalDate datetime) {
-        return datetime.isAfter(expiryDate);
-    }
-
-    /**
-     * Returns {@code true} if both {@link ItemExpiryDate#expiryDate} have the same date by
+     * Returns {@code true} if both {@link ItemExpiryDate} have the same date by
      * {@link LocalDate#equals(Object)}.
      */
     @Override
     public boolean equals(Object other) {
         return other == this
                 || (other != NOT_SET_EXPIRY_DATE
                 && other instanceof ItemExpiryDate
-                && expiryDate.equals(((ItemExpiryDate) other).expiryDate));
-    }
-
-    /**
-     * Compares two item expiry dates. The method returns 0 if the bought date is equal to the other
-     * bought date.
-     * A value less than 0 is returned if the bought date is less than the other bought date (earlier) and
-     * a value greater than 0 if the bought date is greater than the other bought date (later).
-     *
-     * @param other The ItemExpiryDate to compare this ItemExpiryDate against.
-     */
-    public int compareTo(ItemExpiryDate other) {
-        return expiryDate.compareTo(other.expiryDate);
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public int hashCode() {
```

Comment:
I don't think we should remove the `hashCode` method.

Reply:
Okay